index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title id="title">
+            <h1>Survey Form</h1>
+        </title>
+        <meta charset="UTF-8" name="viewport" content="width=device, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+       <p id="description" class="white-text">Holla, Thank  For Visiting Our Survey Page. Please complete the fileds below</p>
+       <hr> <br><br>
+       <form action="" id="survey-form">
+          <fieldset id="box"> 
+            <label id="name-label" for="name">Name:</label><br>
+            <input type="text" id="name" name="name" placeholder="Your name"><br><br>
+
+            <label id="email-label" for="email">Email:</label><br>
+            <input type="text" id="email" name="email" placeholder="Email address"><br><br>
```

Comment:
The input element for the email address should have the type attribute set to "email", which will trigger the email. Kindly fix this to get the form input to work properly